src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -95,6 +95,16 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
         assertCommandSuccess(command, actualModel, expectedCommandResult, expectedModel);
     }
 
+    /**
+     * Convenience wrapper to {@link #assertCommandSuccess(Command, Model, CommandResult, Model)}
+     * that takes a string {@code expectedMessage}.
+     */
+    public static void assertCommandSuccessWithPanelUpdate(Command command, Model actualModel, String expectedMessage,
+            Model expectedModel) {
+        CommandResult expectedCommandResult = new CommandResult(expectedMessage, true);
```

Comment:
I think it would be good to specify `updatePanel` to be `false` for the test above (`assertCommandSuccess`)